Sources/GRPC/GRPCProtobufPayload.swift

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import NIO
+import SwiftProtobuf
+
+public protocol GRPCProtobufPayload: GRPCPayload {}
```

Comment:
`GRPCProtobufPayload` by definition must be a `SwiftProtobuf.Message`, I don't think there's any need to make the extension conditional on that, it might as well be required by the protocol.